src/plugins/formulas/__tests__/formulas.spec.js

```diff
@@ -386,93 +392,66 @@ describe('Formulas general', () => {
       height: 300
     });
 
-    expect(hot.getDataAtCell(0, 1)).toBe('#REF!');
+    expect(hot.getDataAtCell(0, 1)).toBe('#CYCLE!');
   });
 
-  it('should mark cell as #REF! (out of data table range for columns)', () => {
-    const hot = handsontable({
-      data: getDataForFormulas(0, 'name', ['=K1']),
-      columns: getColumnsForFormulas(),
-      formulas: true,
-      width: 500,
-      height: 300
+  // Discussion on why `null` instead of `#REF!` at
+  // https://github.com/handsontable/handsontable/issues/7668
+  describe('out of range cells should return null', () => {
+    it('columns', () => {
```

Comment:
It might be nitpicking, but maybe we can rephrase the test structure descriptions, so it reads more naturally? 
More like:
```
describe('Out of range cells')
// ...
it('should return null for out-of-range columns')
```
or sth?

Same with other similar places in this file.

Reply:
https://github.com/handsontable/handsontable/pull/7648/commits/1289090733b38b6c8451761f64961ae899cce02b